.eslintrc.cjs

```diff
@@ -2,14 +2,12 @@ module.exports = {
   parser: "@typescript-eslint/parser",
```

Comment:
I don't understand why you reformat this file, please revert.

Reply:
Ok..